pkg/wsman/cim/models/constants.go

```diff
@@ -233,201 +403,201 @@ const (
+	CPUStatus_Other
 )
 
 const (
-	OperationalStatusUnknown OperationalStatus = iota
-	OperationalStatusOther
-	OperationalStatusOK
-	OperationalStatusDegraded
-	OperationalStatusStressed
-	OperationalStatusPredictiveFailure
-	OperationalStatusError
-	OperationalStatusNonRecoverableError
-	OperationalStatusStarting
-	OperationalStatusStopping
-	OperationalStatusStopped
-	OperationalStatusInService
-	OperationalStatusNoContact
-	OperationalStatusLostCommunication
-	OperationalStatusAborted
-	OperationalStatusDormant
-	OperationalStatusSupportingEntityinError
-	OperationalStatusCompleted
-	OperationalStatusPowerMode
-	OperationalStatusRelocating
+	OperationalStatus_Unknown OperationalStatus = iota
+	OperationalStatus_Other
+	OperationalStatus_OK
+	OperationalStatus_Degraded
+	OperationalStatus_Stressed
+	OperationalStatus_PredictiveFailure
+	OperationalStatus_Error
+	OperationalStatus_NonRecoverableError
+	OperationalStatus_Starting
+	OperationalStatus_Stopping
+	OperationalStatus_Stopped
+	OperationalStatus_InService
+	OperationalStatus_NoContact
+	OperationalStatus_LostCommunication
+	OperationalStatus_Aborted
+	OperationalStatus_Dormant
+	OperationalStatus_SupportingEntityinError
+	OperationalStatus_Completed
+	OperationalStatus_PowerMode
+	OperationalStatus_Relocating
 )
 
 const (
-	HealthStateUnknown             HealthState = 0
-	HealthStateOK                  HealthState = 5
-	HealthStateDegradedWarning     HealthState = 10
-	HealthStateMinorFailure        HealthState = 15
-	HealthStateMajorFailure        HealthState = 20
-	HealthStateCriticalFailure     HealthState = 25
-	HealthStateNonRecoverableError HealthState = 30
+	HealthState_Unknown             HealthState = 0
+	HealthState_OK                  HealthState = 5
+	HealthState_DegradedWarning     HealthState = 10
+	HealthState_MinorFailure        HealthState = 15
+	HealthState_MajorFailure        HealthState = 20
+	HealthState_CriticalFailure     HealthState = 25
+	HealthState_NonRecoverableError HealthState = 30
 )
 
 const (
-	EnabledStateUnknown EnabledState = iota
-	EnabledStateOther
-	EnabledStateEnabled
-	EnabledStateDisabled
-	EnabledStateShuttingDown
-	EnabledStateNotApplicable
-	EnabledStateEnabledbutOffline
-	EnabledStateInTest
-	EnabledStateDeferred
-	EnabledStateQuiesce
-	EnabledStateStarting
+	EnabledState_Unknown EnabledState = iota
+	EnabledState_Other
+	EnabledState_Enabled
+	EnabledState_Disabled
+	EnabledState_ShuttingDown
+	EnabledState_NotApplicable
+	EnabledState_EnabledbutOffline
+	EnabledState_InTest
+	EnabledState_Deferred
+	EnabledState_Quiesce
+	EnabledState_Starting
 )
 
 const (
-	RequestedStateEnabled RequestedState = iota + 2
-	RequestedStateDisabled
-	RequestedStateShutDown
-	RequestedStateNoChange
-	RequestedStateOffline
-	RequestedStateTest
-	RequestedStateDeferred
-	RequestedStateQuiesce
-	RequestedStateReboot
-	RequestedStateReset
-	RequestedStateNotApplicable
-	RequestedStateUnknown RequestedState = 0
+	RequestedState_Enabled RequestedState = iota + 2
```

Comment:
why is unknown/0 skipped here?

Reply:
These shouldn't be used anymore.  This is why I moved enabled state, requested state into each package because for each class these have different lists of values.

Reply:
this is why these have been moved into each package instead of centralized.  Eventually the constants.go file will go away.